utils/writeable/src/impls.rs

```diff
@@ -6,7 +6,8 @@ use crate::*;
 use core::fmt;
 
 macro_rules! impl_write_num {
-    ($u:ty, $i:ty, $test:ident) => {
+    // random_call exists since usize doesn't have a rand impl. Should always be `random` or empty
```

Comment:
Thought: strange that `usize` doesn't have a rand impl.

Reply:
Yeah we should check upstream